src/main/java/network/brightspots/rcv/CommonDataFormatReader.java

```diff
@@ -203,23 +197,27 @@ CVRContest getCvrContest(CVR cvr, Contest contestToTabulate) {
     CVRContest cvrContestToTabulate = null;
     // find current snapshot
     CVRSnapshot currentSnapshot = null;
-    for (CVRSnapshot cvrSnapshot : cvr.CVRSnapshot) {
-      if (cvrSnapshot.ObjectId.equals(cvr.CurrentSnapshotId)) {
-        currentSnapshot = cvrSnapshot;
-        break;
+    if (cvr.CVRSnapshot != null) {
+      for (CVRSnapshot cvrSnapshot : cvr.CVRSnapshot) {
+        if (cvrSnapshot.ObjectId.equals(cvr.CurrentSnapshotId)) {
+          currentSnapshot = cvrSnapshot;
+          break;
+        }
       }
     }
     // find CVRContest which matches the Contest we are tabulating
-    for (CVRContest cvrContest : currentSnapshot.CVRContest) {
-      if (cvrContest.ContestId.equals(contestToTabulate.ObjectId)) {
-        cvrContestToTabulate = cvrContest;
-        break;
+    if (currentSnapshot.CVRContest != null) {
```

Comment:
This still has issues with `currentSnapshot` potentially being `null` at this point. You can address it by replacing with:
```
if (currentSnapshot != null && currentSnapshot.CVRContest != null) {
```

Reply:
Are you getting these from some style checker or you just have the eagle eye?

Reply:
This one was from `Analyze => Inspect Code...` but the rest were all eagle eyes, baybee!